src/main/java/org/mskcc/cbio/oncokb/web/rest/slack/DropdownEmailOption.java

```diff
@@ -0,0 +1,275 @@
+        .modalSubject(EmailSubject.DEFAULT)
+        .collapsedNote("Clarified with user on multiple account requests")
+        .expandedNote("We have sent a clarification email to the user asking why they registered multiple accounts."))
+    , CLARIFY_REGISTRATION_INFO(new DropdownEmailOptionBuilder()
+        .blockId(BlockId.REGISTRATION_INFO_CLARIFICATION_NOTE)
+        .actionId(ActionId.SEND_REGISTRATION_INFO_CLARIFICATION_EMAIL)
+        .confirmActionId(ActionId.CONFIRM_SEND_REGISTRATION_INFO_CLARIFICATION_EMAIL)
+        .mailType(MailType.CLARIFY_REGISTRATION_INFO)
+        .category(EmailCategory.CLARIFY)
+        .dropdownKey("Send Registration Info Clarification Email")
+        .modalTitle("Clarify Registry Info")
+        .modalSubject(EmailSubject.DEFAULT)
+        .collapsedNote("Clarified with user on registration info")
+        .expandedNote("We have sent a clarification email to the user asking to provide more detailed registration info."))
+    , LICENSE_OPTIONS(new DropdownEmailOptionBuilder()
+        .blockId(BlockId.LICENSE_OPTIONS_NOTE)
+        .actionId(ActionId.SEND_LICENSE_OPTIONS_EMAIL)
+        .confirmActionId(ActionId.CONFIRM_SEND_LICENSE_OPTIONS_EMAIL)
+        .mailType(MailType.LICENSE_OPTIONS)
+        .category(EmailCategory.LICENSE)
+        .specificLicense(LicenseType.COMMERCIAL)
+        .specificLicense(LicenseType.HOSPITAL)
+        .specificLicense(LicenseType.RESEARCH_IN_COMMERCIAL)
+        .dropdownKey("Send License Options Email")
+        .modalTitle("Send License Options")
+        .modalSubject(EmailSubject.COMPANY)
+        .collapsedNote("Sent license options email")
+        .expandedNote("We have sent an email to the user with license options for their affiliated company."))
+    , REJECT(new DropdownEmailOptionBuilder()
+        .blockId(BlockId.REJECTION_NOTE)
+        .actionId(ActionId.SEND_REJECTION_EMAIL)
+        .confirmActionId(ActionId.CONFIRM_SEND_REJECTION_EMAIL)
+        .mailType(MailType.REJECTION)
+        .category(EmailCategory.DENY)
+        .dropdownKey("Send Rejection Email")
+        .modalTitle("Rejection Email")
+        .modalSubject(EmailSubject.DEFAULT)
+        .collapsedNote("Sent rejection email")
+        .expandedNote("The user has been rejected and notified."))
+    , REJECT_ALUMNI_ADDRESS(new DropdownEmailOptionBuilder()
+        .blockId(BlockId.REJECT_ALUMNI_ADDRESS_NOTE)
+        .actionId(ActionId.SEND_REJECT_ALUMNI_ADDRESS_EMAIL)
+        .confirmActionId(ActionId.CONFIRM_SEND_REJECT_ALUMNI_ADDRESS_EMAIL)
+        .mailType(MailType.REJECT_ALUMNI_ADDRESS)
+        .category(EmailCategory.DENY)
+        .dropdownKey("Send Alumni Rejection Email")
+        .modalTitle("Reject Alumni Address")
+        .modalSubject(EmailSubject.DEFAULT)
+        .collapsedNote("Rejected user due to alumni email address")
+        .expandedNote("The user has been rejected due to alumni email address."))
+    ;
+
+
+    private static class DropdownEmailOptionBuilder {
```

Comment:
Can we add comments on which fields are required? Probably even better to include the required fields(hopefully not that many) in the contractor.